### PR TITLE
chore(contributor): update contributor guidelines with more information

### PR DESCRIPTION
**Pull request by nellyk**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,145 +1,121 @@
...
-#### Type
+3. Install the dependencies by running
 
-The *type* should always be lowercase as shown below.
+    ```bash
+    $ cd npm install
+    ```
 
-##### Allowed `<type>` values:
+4. You can now run any of this scripts from the root folder.
```
Comment (anescobar1991) [inline CONTRIBUTING.md]: `this` should be `these`.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,145 +1,121 @@
...
-```
-and for Pull Requests, use the format:
+Provide the system information which is not limited to the below:
 
-```bash
-Related #37
-```
+- OS: [e.g. macOS, Windows]
+- Browser (if applies) [e.g. chrome, safari]
+- Version of faux-cdn: [e.g. 5.0.0]
```
Comment (anescobar1991) [inline CONTRIBUTING.md]: This should be `eslint-config-amex`.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,145 +1,121 @@
...
-<type>(<scope>): <subject>
-```
+   > replace `your-github-username` with your github username
 
-#### Type
+3. Install the dependencies by running
 
-The *type* should always be lowercase as shown below.
+    ```bash
+    $ cd npm install
```
Comment (10xLaCroixDrinker) [inline CONTRIBUTING.md]:
```suggestion
    $ npm install
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,145 +1,121 @@
...
-*Example*: Transactions may be shown in multiple routes/contexts, but a bug affecting transaction actions may only exist under the "home" route, possibly related to other code. In such cases, you could use the following format:
+Runs linting on the current branch and checks that the commits follow [conventional commits](https://www.conventionalcommits.org/)
 
-```bash
-fix(home/transactions): txn actions not working
-```
+## Submitting a new feature
 
-This header makes it clear that the fix is limited in scope to transactions within the home route/context.
+When submitting a new feature request or enhancement of an existing features please review the following:-
```
Comment (10xLaCroixDrinker) [inline CONTRIBUTING.md]:
```suggestion
When submitting a new feature request or enhancement of an existing features please review the following:
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,145 +1,121 @@
...
-Closes #17, #26
-```
-and for Pull Requests, use the format:
+Provide the system information which is not limited to the below:
 
-```bash
-Related #37
-```
+- OS: [e.g. macOS, Windows]
+- Browser (if applies) [e.g. chrome, safari]
```
Comment (10xLaCroixDrinker) [inline CONTRIBUTING.md]:
```suggestion
```

**Pull request merged**